frame/App/Body/Main/Layout/components/ListItem.js

```diff
@@ -36,6 +38,7 @@ const ListItem = ({
     container: {
       borderTop: '1px solid #cccccc',
       textDecoration: completed ? 'line-through' : 'initial',
+      backgroundColor: priority ? 'grey' : null,
```

Comment:
In JS there's `undefined` and `null` they mean kinda the same thing but are different. As a general rule I would say use `undefined` unless it's a special case that NEEDS `null`. React components is one case where you'd write `return null`. Otherwise just stick with `undefined` and your life will have no issues. 
```suggestion
      backgroundColor: priority ? 'grey' : undefined,
```